packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -38,13 +37,6 @@ export const EventMeta = () => {
   const isEmbed = useIsEmbed();
   const hideEventTypeDetails = isEmbed ? embedUiConfig.hideEventTypeDetails : false;
 
-  useEffect(() => {
-    if (!selectedDuration && event?.length) {
-      setSelectedDuration(event.length);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [event?.length, selectedDuration]);
```

Comment:
Was causing the whole booker to re-render. not ideal. Shallow doesnt work as expected here